app/src/main/java/org/mozilla/reference/browser/EngineProvider.kt

```diff
@@ -40,7 +41,11 @@ object EngineProvider {
 
     fun createEngine(context: Context, defaultSettings: DefaultSettings): Engine {
         val runtime = getOrCreateRuntime(context)
-        return GeckoEngine(context, defaultSettings, runtime)
+
+        val engine = GeckoEngine(context, defaultSettings, runtime)
+        WebCompatFeature.install(engine)
+
+        return engine
```

Comment:
This time no usage of `also`? :)

Reply:
The old code already assigned the runtime to a variable, so I just kept this style for consistency. But I'm not opposed to change it, let me know! :)

Reply:
Hm, I changed it. :)